create-addons-loader.js

```diff
@@ -67,14 +68,18 @@ const safeWrapper = (func) => (config) => {
   return res;
 }
 
+const projectConfig = (config) => {
+  return typeof projectConfigLoader.default === "function" ? projectConfigLoader.default(config) : config;
```

Comment:
@tiberiuichim wondering if we could squeeze this point even a bit more and lookup for a `volto.config.js` too, and if it exists, then load its configuration too. It could be useful for (cypress) testing...